### DISCUSSION TITLE
Product feed to use with Google Shopping and others

### DISCUSSION THREAD

**Discussion by mrguruo** [medusajs/medusa#4717] [Feature Requests]
There doesn't appear to be a way to create a product feed for use with shopping aggregators such as Google Shopping. This functionality is important for a lot of stores.

More information from Google here:
https://support.google.com/merchants/answer/160567

**Comment by tsvetann** [comment]
Isn't this something your frontend should be able to generate as it is decoupled from medusa

**Comment by mattwilliams150** [comment]
is this a feature yet?

**Comment by redareda9** [comment]
Would be a nice plugin, I'm migrating from Shopify and this is the only thing making me procrastinate.

**Comment by henry9207** [comment]
Any news?

**Comment by ngethe** [comment]
I can't find one it seems I will have to create a it

**Comment by harakiro** [comment]
I've begun building this plugin, It should be something from the backend not the storefront. Will share updates as I get this created.

**Comment by zCastleM** [comment]
This shouldn't work?
https://github.com/fsyntax/medusa-plugin-feeds

**Comment by radeknapora** [comment]
https://docs.medusajs.com/resources/how-to-tutorials/tutorials/product-feed
